### PR TITLE
Add 2017 schedule, speaker, and talk pages

### PR DESCRIPTION
**Pull request by juliusv**
This also removes the diversity sponsorship application form, as it is
now too late to sponsor more applicants.

### CODE REVIEW

**Comment by juliusv** [comment]
@brian-brazil

File: content/2017-munich/schedule.md [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+    <td>18:00</td>
+    <td>Break</td>
+    <td></td>
+  </tr>
+  <tr class="break">
+    <td>18:30</td>
+    <td>
+      Happy Hour (details to be announced)
+      <br>
+      Open End
```
Comment (brian-brazil) [inline content/2017-munich/schedule.md]: I'd suggest "Til Late"

File: content/2017-munich/schedule.md [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+        Lightning Talks
+      </a>
+    </td>
+    <td></td>
+  </tr>
+  <tr class="talk">
+    <td>18:00</td>
+    <td>
+      <a href="/2017-munich/talks/closing">
+        Closing
```
Comment (brian-brazil) [inline content/2017-munich/schedule.md]: Should we have an end time on this?

File: content/2017-munich/speakers/tom-wilkie.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+title: Tom Wilkie
+---
+
+## Tom Wilkie
+
+Tom is a Software Engineer at Weaveworks. Previously he was at Google as a Site Reliability Manager for Google Analytics. Before that he was Founder, VP Eng and CTO at Acunu, and before that a Software Engineer at XenSource. In his spare time, Tom likes to make craft beer and build home automation systems.
```
Comment (brian-brazil) [inline content/2017-munich/speakers/tom-wilkie.md]: @tomwilkie - do you wish to change this?

**Comment by tomwilkie** [comment]
Yes please! Will do it tomorrow.

File: content/2017-munich/schedule.md (hunk 1 above)
Comment (juliusv) [inline content/2017-munich/schedule.md]: I'll go with the less vernacular form "Until Late".

**Comment by juliusv** [comment]
Made some updates, including setting Tom's bio to something agreed upon in chat, and making it clear that the bios for Carl and RichiH are still to follow.

**Comment by juliusv** [comment]
Timing-wise, good to merge now? We can still use a better time slot to publicize it more, but we're quite late already, so I wouldn't want to wait until Monday for the merge just for PR.

**Comment by brian-brazil** [comment]
I already gave the :+1:  :)

**Comment by juliusv** [comment]
Ok :)

**Pull request merged**